app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  SYSTEM_ID = 1
```

Comment:
weird. was this required by state machine? silly @austinkahly musta not carved it out very well...